.github/workflows/notify-slack-on-release-merge.yml

```diff
@@ -0,0 +1,68 @@
+name: Slack notifications for releases/* merges
+
+# This CI job will push notifications to Slack whenever code is merged into any of the
+# release branches.
+
+on:
+  push:
+    branches:
+      - 'releases/*'
+
+jobs:
+  notify-slack:
+    name: 'Emit Slack notification(s)'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Parse event to Slack messages
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: |
+          echo $GITHUB_CONTEXT | jq '.event.commits[] | [.url, .id[0:7], .author.username, .timestamp, .message] | @tsv' | sed 's/"//g' | sed 's/\\t/;/g' | sed 's/\\n/;/g' | sed 's/\\//g' | awk -F';' '{print "• <"$1"|"$2"> (<https://github.com/"$3"|"$3">, "$4") - "$5}' | sed 's/$/\\n/g' | tr -d '\n' > /tmp/foo
```

Comment:
Why do this inline? Would it be more readable if you called out to a script?
If you leave this inline, can you break it into a multi-line statement with comments?

Reply:
I don't recall the context but inline was one of the only ways I could get this to work when I was iterating on this for the other repository where it originated. It could be that a shell script would work as well _but_ in the interest of getting the CI pipeline working my preference would be to add comments here explaining the commands rather than rework anything.